src/Analysis/GWASResults/SharedComponents/Diagrams/lib/pheweb_plots.js

```diff
@@ -0,0 +1,608 @@
+        else {
+            possible_ticks = range(0, 20.1, 4);
+            if (max_data_qval <= 70) { possible_ticks = possible_ticks.concat([30,40,50,60,70]); }
+            else if (max_data_qval <= 120) { possible_ticks = possible_ticks.concat([40,60,80,100,120]); }
+            else if (max_data_qval <= 220) { possible_ticks = possible_ticks.concat([60,100,140,180,220]); }
+            else {
+                const power_of_ten = 10 ** Math.floor(Math.log10(max_data_qval));
+                const first_digit = max_data_qval / power_of_ten;
+                let multipliers;
+                if (first_digit <= 2) { multipliers = [0.5, 1, 1.5, 2]; }
+                else if (first_digit <= 4) { multipliers = [1, 2, 3, 4]; }
+                else { multipliers = [2, 4, 6, 8, 10]; }
+                possible_ticks = possible_ticks.concat(multipliers.map(function(m) { return m * power_of_ten; }));
+            }
+        }
+        // Include all ticks < qval.  Then also include the next tick.
+        // That should mean we'll always have the largest tick >= the largest variant.
+        const ticks = possible_ticks.filter(function(qval) { return qval < max_data_qval; });
+        if (ticks.length < possible_ticks.length) { ticks.push(possible_ticks[ticks.length]); }
+
+        // Use the largest tick for the top of our y-axis so that we'll have a tick nicely rendered right at the top.
+        let max_plot_qval = ticks[ticks.length - 1];
+        // If we have any qval=inf (pval=0) variants, leave space for them.
+        if (includes_pval0) { max_plot_qval *= 1.1; }
+        let scale = d3Scale.scaleLinear().clamp(true);
+        if (max_plot_qval <= 40) {
+            scale = scale
+                .domain([max_plot_qval, 0])
+                .range([0, plot_height]);
+        } else {
+            scale = scale
+                .domain([max_plot_qval, 20, 0])
+                .range([0, plot_height / 2, plot_height]);
+        }
+
+        if (includes_pval0) { ticks.push(Infinity); }
+
+        return {
+            'scale': scale,
+            'draw_break_at_20': !(max_plot_qval <= 40),
+            'ticks': ticks,
+        };
+    }
+
+    window.addEventListener('load', function() {
+        const svg_width = document.getElementById('manhattan_plot_container').offsetWidth;
+        const svg_height = 550;
+        const plot_margin = {
+            'left': 70,
+            'right': 30,
+            'top': 20,
+            'bottom': 50,
+        };
+        const plot_width = svg_width - plot_margin.left - plot_margin.right;
+        const plot_height = svg_height - plot_margin.top - plot_margin.bottom;
+
+        const gwas_svg = d3.select('#manhattan_plot_container').append('svg')
+            .attr('id', 'gwas_svg')
+            .attr('width', svg_width)
+            .attr('height', svg_height)
+            .style('display', 'block')
+            .style('margin', 'auto');
+        const gwas_plot = gwas_svg.append('g')
```

Comment:
If we want to improve accessibility by adding a SVG ```<details>``` to hold assistive text we could likely do it here, something like: 

```
gwas_svg.append('details').text(()=> 
'A chart showing a pheweb visualization. The data is also presented in the table below ');
```
Validated that this works:
<img width="797" alt="image" src="https://github.com/uc-cdis/data-portal/assets/113449836/78468e68-1abf-427e-a346-7628732edf90">

Reply:
done.